numpy/lib/tests/test_nanfunctions.py

```diff
@@ -588,6 +588,15 @@ def test_empty(self):
                     assert_(len(w) == 0)
 
 
+_TIME_UNITS = (
+    "Y", "M", "W", "D", "h", "m", "s", "ms", "us", "ns", "ps", "fs", "as"
+)
```

Comment:
I don't think we a concrete list of time units exposed anywhere inside numpy, do we?
Might be worthwhile to add at some point.